lib/routes/account.js

```diff
@@ -52,6 +52,7 @@ module.exports = function (
         var authSalt = crypto.randomBytes(32)
         var authPW = Buffer(form.authPW, 'hex')
         var locale = request.app.acceptLanguage
+        var userAgent = request.headers['user-agent']
```

Comment:
We use ua-parser here https://github.com/mozilla/fxa-content-server/blob/master/package.json#L71  
see https://www.npmjs.com/package/ua-parser